src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxRandomBalls = 100;

```

Comment:
```suggestion
    private static final int MAX_BALL_NUMBER = 100;

```

https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names